apps/chat-e2e/src/tests/sharedFilesAttachments.test.ts

```diff
@@ -0,0 +1,477 @@
+          secondImageEntity,
+          Colors.controlsBackgroundAccent,
+        );
+
+        const thirdImageEntity: TreeEntity = {
+          name: Attachment.flowerImageName,
+        };
+        await attachedFilesAssertion.assertSharedFileArrowIconState(
+          thirdImageEntity,
+          'visible',
+        );
+        await attachedFilesAssertion.assertEntityArrowIconColor(
+          thirdImageEntity,
+          Colors.controlsBackgroundAccent,
+        );
+
+        //TODO EPMRTC-4135 blocked by the #1076
+        // const fourthImageEntity: TreeEntity = { name: Attachment.heartImageName };
+        // await attachedFilesAssertion.assertSharedFileArrowIconState(fourthImageEntity, 'visible');
+        // await attachedFilesAssertion.assertEntityArrowIconColor(fourthImageEntity, Colors.controlsBackgroundAccent);
+
+        const specialCharsImageEntity: TreeEntity = {
+          name: Attachment.specialSymbolsName,
+        };
+        await attachedFilesAssertion.assertSharedFileArrowIconState(
+          specialCharsImageEntity,
+          'visible',
+        );
+        await attachedFilesAssertion.assertEntityArrowIconColor(
+          specialCharsImageEntity,
+          Colors.controlsBackgroundAccent,
+        );
+        await attachFilesModal.closeButton.click();
+      },
+    );
+
+    await dialSharedWithMeTest.step(
+      'Prepare conversation to share of the user 2',
+      async () => {
+        additionalShareUserConversationData.resetData();
+        const conversationToShare =
+          additionalShareUserConversationData.prepareEmptyConversation(
+            ModelIds.GPT_4_O,
+          );
+        await additionalShareUserDataInjector.createConversations([
+          conversationToShare,
+        ]);
+        await additionalShareUserLocalStorageManager.setSelectedConversation(
+          conversationToShare,
+        );
+      },
+    );
+
+    await dialSharedWithMeTest.step(
+      'By user2 create a conversation with attachments from Shared with me section in Manage attachments',
+      async () => {
+        const newRequest = GeneratorUtil.randomString(10);
+        await additionalShareUserDialHomePage.openHomePage();
+        await additionalShareUserDialHomePage.waitForPageLoaded({});
+        await additionalShareUserSendMessage.attachmentMenuTrigger.click();
+
+        await additionalShareUserAttachmentDropdownMenu.selectMenuOption(
+          UploadMenuOptions.attachUploadedFiles,
+        );
+
+        await additionalShareUserAttachFilesModal.checkAttachedFile(
+          Attachment.specialSymbolsName,
+          FileModalSection.SharedWithMe,
+        );
+        await additionalShareUserAttachFilesModal.attachFiles();
+        await additionalShareUserDialHomePage.mockChatTextResponse(MockedChatApiResponseBodies.simpleTextBody);
+        await additionalShareUserChat.sendRequestWithButton(newRequest);
+        await additionalShareUserChat.waitForResponse();
```

Comment:
response waiting is called inside the `sendRequestWithButton()` method

Reply:
done